src/app/lib/analyticsUtils/index.test.js

```diff
@@ -387,3 +388,23 @@ describe('getPublishedDatetime', () => {
     expect(publishedTime).toEqual(null);
   });
 });
+
+describe('getAtUserId', () => {
+  returnsNullWhenOffClient(getAtUserId);
```

Comment:
I think this is missing:
```suggestion
  returnsNullWhenOffClient(getAtUserId());
```